scripts/test/nightly/runTests.sh

```diff
@@ -34,13 +34,11 @@ VPC=$2
 SUBNETS=$3
 RESULTS_BUCKET=$4
 MAIN_SUITE_NAME=$5
+SUITE_PARAMS=("-Dsleeper.system.test.cluster.enabled=true" "-DskipRust" "-Dsleeper.system.test.create.multi.platform.builder=false")
```

Comment:
MAIN_SUITE_PARAMS is still used in a few places. Maybe we should make it consistent so we always use SUITE_PARAMS instead of MAIN_SUITE_PARAMS?